src/rwrapr/renv.py

```diff
@@ -234,6 +236,59 @@ def reval(self, expr: str, rview: bool | None = None) -> Any:
         else:
             return convert_r2py(r_object)
 
+    def rscript(
+        self,
+        path: str | None = None,
```

Comment:
I suggest that you change to use pathlib.Path as parameter instead of str to handle paths. This is recommended in modern python, see [this article on medium.com](https://medium.com/@guandika8/stop-representing-file-paths-in-python-using-strings-8312791f10f) and [this article on RealPython](https://realpython.com/python-pathlib/#the-problem-with-representing-paths-as-strings).

Reply:
`path` can now be supplied using a `pathlib.Path` object. `path` can however still be `str`. If `path` is `str`, `path` is converted to `pathlib.Path` before attempting to open the file.